package.json

```diff
@@ -45,6 +45,7 @@
     "install:docs": "(cd docs && npm install --legacy-peer-deps)",
     "lint": "eslint '**/*.{js,ts,tsx,md,mdx}' --max-warnings=0",
```

Comment:
We have a set of rules for the `md` and `mdx` files already [on the eslint config.](https://github.com/primer/react/blob/main/.eslintrc.json#L125) I am not sure if there would be any conflict between them? Or what do you suggest for us to do here?

Reply:
oooh interesting! I don't think `eslint` runs the same rulesets as markdownlint. do you know what rules are running on markdown?